Data/Uniques/gloves.lua

```diff
@@ -383,6 +383,16 @@ Requires Level 27, 22 Str, 22 Dex
 {variant:2}+40% to Cold Resistance
 You gain Onslaught for 5 seconds on using a Vaal Skill
 ]],[[
+Worldcarver
+Dragonscale Gauntlets
+Requires Level 67, 51 Str, 51 Dex
+Trigger Level 20 Arcane Wake after Spending a total of 200 Mana
++(200–300) to Accuracy Rating
+(120–150)% increased Armour and Evasion
++(60–80) to maximum Life
+3% increased Attack Speed for each Map Item Modifier affecting the Area
```

Comment:
According to pathofexile.com/trade Attack Speed should be after Attack Damage. Wiki probably has it wrong.

Reply:
Yup. Trade has speed after damage. Poedb had it the other way around. I'll change it anyways.